reference/examples/TransferTokensUsingPublicAddress.md

```diff
@@ -0,0 +1,13 @@
+
+Transfer HBAR or tokens to a Hedera account using their public-address.
+
+Reference: HIP-583
+
+## Example 1
+- Create a Ethereum Account Address / public-address	(Example: 0xb794f5ea0ba39494ce839613fffba74279579268)
+- Transfer tokens using the `TransferTransaction` to the Etherum Account Address
```

Comment:
What account/key/address is signing the transfer transaction?

Reply:
This is not a supported scenario. A long zero maps to an accountID. You can only create a hollow account using the evm address / public address